apps/playground/src/app/modules/components/toggle-module/toggle-module.component.scss

```diff
@@ -0,0 +1,36 @@
+// Imports
+@use "variables" as var;
+
+// Actual styling
+.module-simulations {
+  margin-top: 2 * var.$default-margin;
+
+  .mat-card {
+    margin: var.$default-margin 0 2 * var.$default-margin;
+
+    .simulation {
+      display: flex;
+      flex-direction: column;
+      padding: var.$default-padding;
+
+      & > * {
+        flex: 1;
+      }
+
+      .settings {
+        margin-right: var.$default-margin;
+
+        form {
+          display: flex;
+          flex-direction: column;
+          align-items: flex-start;
+
+          button {
+            margin: var.$default-margin;
+            align-self: flex-end;
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
Also, the new version for `apps/playground/src/app/modules/components/simulator/simulator.component.scss`
```css
// Imports
@use "variables" as var;

// Actual styling
div.content-wrapper {
  border-radius: 2px;
  height: 100%;
  padding: var.$default-padding;
  overflow: visible; // if it is 'hidden', we do not see the dropdown menu of the navigation component.

  /*
   * Adding this to create a new stacking context and
   * completely isolate the CSS within the simulator
   */
  transform: translate(0, 0);
}
```